### ISSUE TITLE
Severe influenza infection

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7456]
**Preferred term label:**
Severe influenza infection

**Synonyms**

**Definition (free text, please give PubMed ID)**
An unusually severe infection by a parainfluenza virus.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:616345 
PMID:25814066 1/1

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
